central/features/org.jboss.tools.central.feature/feature.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<feature id="org.jboss.tools.central.feature" label="%featureName" version="2.1.300.qualifier" provider-name="%providerName" plugin="org.jboss.tools.central"
+<feature id="org.jboss.tools.central.feature" label="%featureName" version="2.1.301" provider-name="%providerName" plugin="org.jboss.tools.central"

```

Comment:
What's the reason for the qualifier to be removed ?

Reply:
thx for spotting, my wrong. Correcting it.